### DISCUSSION TITLE
How do I post my OpenWeatherMap info to the "customize wallpaper" section?

### DISCUSSION THREAD

**Discussion by ghost** [rocksdanister/lively#244] [Q&A]
Under the "customize wallpaper" bar for the "ripples" background, it doesn't give specifics on the formatting for the API key, city, and country. For API key, does it just want the key? Or the key in contained in a URL? For the location, does the link to my city's OpenWeatherMap page? There's no documentation I could find on this anywhere.

**Comment by rocksdanister** [comment]
There is no documentation because it is a wallpaper and not part of Lively program.
Just the key and city name is enough (The default text on the input box should be a hint for the formatting.)

Original page has some instructions:
https://www.deviantart.com/livelywallpaper/gallery/71975121/customisable-web-wallpapers

**Comment by erwinwallpaper** [comment]
Thank You for the instructions.
